### ISSUE TITLE
Calling `lib.importJSON` Fails When `_github.enable` is False

### ISSUE THREAD

**Issue by mnaser** [mirkolenz/flocken#80]
When `_github.enable` is set to false, the current code in `docker-manifest.nix` evaluates `lib.optionalAttrs`, which results in an empty set (`{}`). This causes the subsequent call to `lib.importJSON` to fail because it attempts to call the empty set as a function.

```
       error: attempt to call something which is not a function but a set: { }
       at /nix/store/4cnxks7ly9d9f3fi27l1fdp099dlr147-source/src/docker-manifest.nix:44:16:
           43|
           44|   githubData = lib.optionalAttrs (_github.enable) lib.importJSON (
             |                ^
           45|     builtins.fetchurl "${_github.apiEndpoint}/repos/${_github.repo}"
```

https://github.com/mirkolenz/flocken/blob/main/src/docker-manifest.nix#L44-L46

What would be the ideal fix?

**Comment by mirkolenz** [comment]
Thanks for the bug report! I think it's caused by missing parentheses around the json import statement. I just pushed the changes, could you check if this is fixed with v2.4.1? If not, I'll do some additional investigations!

**Comment by mnaser** [comment]
@mirkolenz Thanks!  It seems that I went past that, I've got another issue now which is that my local system cannot emulate aarch64, but that's another problem/challenge that's unrelated to this issue. :)
